ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -150,12 +151,38 @@ export const getUpdatedProfileValue = (
   }
 };
 
+export const isCompleted = (
+  profile: Profile,
+  moduleName: AccessModule
+): boolean =>
+  !!getAccessModuleStatusByName(profile, moduleName)?.completionEpochMillis;
+
 export const isBypassed = (
   profile: Profile,
   moduleName: AccessModule
 ): boolean =>
   !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
 
+export const isExpired = (
+  profile: Profile,
+  moduleName: AccessModule
+): boolean =>
+  !!getAccessModuleStatusByName(profile, moduleName)?.expirationEpochMillis;
+
+export const displayModuleStatus = (
+  profile: Profile,
+  moduleName: AccessModule
+): string =>
+  bypassedOrCompleteAndNotExpiring(
```

Comment:
We need to use the same logic as the individual modules' displays in AAR.  It's going to be much easier to keep that in sync if we reuse common logic instead of replicating it.

See `computeRenewalDisplayDates()` for the specific decision making used for the individual modules in AAR.  I bet we can extract some common logic between here and AAR.  Maybe it would make sense to use an enum for the possible states here?

`bypassedOrCompleteAndNotExpiring()` is used in AAR to summarize the modules, not set individual module text